# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1791883919 -7200
# Parent  e4c4fac934d7b16a352ec2e678e27a08364a2091
Release: Set version for release

!!!                                                       !!!
!!! Do not forget to run ./build.sh NNN delres on Windows !!!
!!!                                                       !!!

diff --git a/.gecko_rev.yml b/.gecko_rev.yml
--- a/.gecko_rev.yml
+++ b/.gecko_rev.yml
@@ -1,9 +1,9 @@
 ---
 GECKO_BASE_REPOSITORY: https://hg.mozilla.org/mozilla-unified
 GECKO_HEAD_REPOSITORY: https://hg.mozilla.org/releases/mozilla-esr153
 GECKO_HEAD_REF: FIREFOX_153_5_0esr_BUILD1
-# GECKO_HEAD_REV: e98cdedc4ba38eb7539595157cdd9c9cedac44fa
+GECKO_HEAD_REV: 3806652efdaccfd958d04803eddd3fdf131cd0cd
 
 ######
 
 ### For comm-central
diff --git a/mail/config/version_display.txt b/mail/config/version_display.txt
--- a/mail/config/version_display.txt
+++ b/mail/config/version_display.txt
@@ -1,1 +1,1 @@
-153.5.0esr
+153.5.0esr-bb11
